scripts/make_evalai_json.py

```diff
@@ -0,0 +1,39 @@
+import json
+import os
+
+
+def main(paths, filename, data_split="val"):
+    submission_file = {}
+
+    for idx, split in enumerate(["id", "ood_ads", "ood_bulk", "ood_ads_bulk"]):
+        key = "_".join([data_split, split])
+        submission_file[key] = json.load(open(os.path.join(paths[idx]), "r"))
+
+    with open(filename, "w") as f:
+        json.dump(submission_file, f)
+
+
+if __name__ == "__main__":
+    """
+    Create a submission file for evalAI. Ensure that for the task you are
+    submitting for you have generated results files on each of the 4 splits -
+    id, ood_ads, ood_bulk, ood_ads_bulk.
+
+    Results file can be obtained as follows for the various tasks:
+
+    S2EF: config["mode"] = "predict"
+    IS2RE: config["mode"] = "predict"
+    IS2RS: config["mode"] = "predict and "config["task"]["write_pos"] = True
```

Comment:
Shouldn't the mode here be "run_relaxations"?

Reply:
Good catch - fixed.